src/barplot.ts

```diff
@@ -34,6 +34,15 @@ export interface IBarPlotOptions extends IVisInstanceOptions {
    * @default NaN
    */
   max?: number;
+  /**
+   * scale such that the height matches the argument
+   * @default null
+   */
+  heightTo?: number;
+  /**
+   * @default 10
```

Comment:
How can that be `10` if it is dependend on the `heightTo` and data dimension in line 66?